Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'pry', '~> 0.10', group: :development, require: false
 gem 'aruba', '~> 0.7.4', require: false
 gem 'rspec', '~> 3.0', require: false
 gem 'cucumber', '~> 2.0', require: false
+gem 'capybara', require: false # middleman-core forces all plugins to declare this
```

Comment:
Doesn't look like it helped. I think the problem is that something requires `cucumber/capybara`.